components/gfx/platform/freetype/font_list.rs

```diff
@@ -207,226 +208,23 @@ pub fn fallback_font_families(options: FallbackFontSelectionOptions) -> Vec<&'st
-            UnicodeBlock::Carian => families.push("Noto Sans Carian"),
-            UnicodeBlock::CaucasianAlbanian => families.push("Noto Sans Caucasian Albanian"),
-            UnicodeBlock::Chakma => families.push("Noto Sans Chakma"),
-            UnicodeBlock::Cham => families.push("Noto Sans Cham"),
-            UnicodeBlock::Cherokee | UnicodeBlock::CherokeeSupplement => {
-                families.push("Noto Sans Cherokee")
-            },
-            UnicodeBlock::Coptic => families.push("Noto Sans Coptic"),
-            UnicodeBlock::Cuneiform | UnicodeBlock::CuneiformNumbersandPunctuation => {
-                families.push("Noto Sans Cuneiform")
-            },
-            UnicodeBlock::CypriotSyllabary => families.push("Noto Sans Cypriot"),
-            UnicodeBlock::Deseret => families.push("Noto Sans Deseret"),
-            UnicodeBlock::Devanagari |
-            UnicodeBlock::DevanagariExtended |
-            UnicodeBlock::CommonIndicNumberForms => families.push("Noto Sans Devanagari"),
-            UnicodeBlock::Duployan => families.push("Noto Sans Duployan"),
-            UnicodeBlock::EgyptianHieroglyphs => families.push("Noto Sans Egyptian Hieroglyphs"),
-            UnicodeBlock::Elbasan => families.push("Noto Sans Elbasan"),
-            UnicodeBlock::Ethiopic |
-            UnicodeBlock::EthiopicExtended |
-            UnicodeBlock::EthiopicExtendedA |
-            UnicodeBlock::EthiopicSupplement => families.push("Noto Sans Ethiopic"),
-            UnicodeBlock::Georgian | UnicodeBlock::GeorgianSupplement => {
-                families.push("Noto Sans Georgian")
-            },
-            UnicodeBlock::Glagolitic | UnicodeBlock::GlagoliticSupplement => {
-                families.push("Noto Sans Glagolitic")
-            },
-            UnicodeBlock::Gothic => families.push("Noto Sans Gothic"),
-            UnicodeBlock::Grantha => families.push("Noto Sans Grantha"),
-            UnicodeBlock::Gujarati => families.push("Noto Sans Gujarati"),
-            UnicodeBlock::Gurmukhi => families.push("Noto Sans Gurmukhi"),
-            UnicodeBlock::HangulCompatibilityJamo |
-            UnicodeBlock::HangulJamo |
-            UnicodeBlock::HangulJamoExtendedA |
-            UnicodeBlock::HangulJamoExtendedB |
-            UnicodeBlock::HangulSyllables => {
-                families.push("Noto Sans KR");
-                families.push("Noto Sans CJK KR");
-            },
-            UnicodeBlock::Hanunoo => families.push("Noto Sans Hanunoo"),
-            UnicodeBlock::Hatran => families.push("Noto Sans Hatran"),
-            UnicodeBlock::Hebrew => families.push("Noto Sans Hebrew"),
+            UnicodeBlock::EnclosedIdeographicSupplement => families.push("WenQuanYi Micro Hei"),
             UnicodeBlock::Hiragana |
             UnicodeBlock::Katakana |
             UnicodeBlock::KatakanaPhoneticExtensions => {
                 families.push("TakaoPGothic");
```

Comment:
`add_noto_fallback_families` [already](https://github.com/servo/servo/pull/32463/files#diff-5ea622dafd8b31eadec41754d816ec902261b3e35293a5d8e62426eb1cbad6b0R174) adds `TakaoPGothic` to the list. Was the intention to remove that line from `add_noto_fallback_families` and add the non-Noto families here? If so, wouldn't this change the order of fallback i.e  `Noto Sans JP` and `Noto Sans CJK JP` would appear before `TakaoPGothic` in the fallback list. Is that fine?

Reply:
Whoops! Nice catch. These lists are so long it is easy to make a mistake. I've removed the duplicate addition. It's true that this will now add `TakaoPGothic` after the Noto fonts, but I think that's okay. The Noto fonts are high-quality and it's likely just fine if those are used as a fallback before other ones. This isn't the case for Mac, which ships with high-quality non-Noto fonts that should likely be selected first.